task/04-arrays-tasks.js

```diff
@@ -519,6 +630,13 @@ function group(array, keySelector, valueSelector) {
  *   ['one','two','three'], x=>x.split('')  =>   ['o','n','e','t','w','o','t','h','r','e','e']
  */
 function selectMany(arr, childrenSelector) {
+   let arr1 =  arr.map(function(item){
+      return childrenSelector(item);
+   })
+   let arr2 =  arr1.reduce(function(a, b){ 
+         return a.concat(b);
+     });
+   return arr2;
     throw new Error('Not implemented');
```

Comment:
you don't need to declare new variables, you can make it easier:
`return arr.map(...).reduce(...);`